03-amazon_s3.Rmd

```diff
@@ -0,0 +1,244 @@
+# Amazon S3
+
+## What is Amazon S3?
+
+[Amazon S3](https://aws.amazon.com/s3/) is a web-based cloud storage platform. It is one of the primary file storage locations on the Analytical Platform, alongside individual users' home directories.
```

Comment:
That's all true. I think you do need to say it stores files in the cloud, as you've done here. But I'd prefer the emphasis was on the philosophy of S3 being the place where we store all data used and outputs of the analysis. A user's Home directory is their working area, with temporary copies of their code, data and outputs, but the main copies of these should be held on GitHub (code) and S3 (data & results) so that you can collaborate and share them.

Reply:
Added in 26514c9.